ci/meta.yml

```diff
@@ -144,3 +144,117 @@ meta:
           dir: concourse-smoke-tests
         inputs:
           - name: terraform-output
+    generate-positive-proxy-test-manifest:
```

Comment:
You could make this one task and set two env vars for the different values and use them. Not blocking comment, but would make code a bit simpler.

Reply:
We could.  But we shamelessly stole this from elsewhere...